app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -376,16 +376,36 @@ def host_switches(extra_attributes = {})
       attributes.merge!(extra_attributes)
     end
 
-    def switches(extra_attributes = {})
+    def host_virtual_switches(extra_attributes = {})
       attributes = {
-        :model_class                 => ::Switch,
+        :model_class                 => Switch,
+        :manager_ref                 => %i(host uid_ems),
```

Comment:
@Ladas this one is giving me problems because there isn't a host_id on switches

Reply:
hm, yeah we would need to add that, or modify the uid_ems.

Reply:
Okay I switched this and https://github.com/ManageIQ/manageiq-providers-vmware/pull/300 to use switch.host_id